src/PrestaShopBundle/Translation/Api/StockApi.php

```diff
@@ -41,6 +41,7 @@ public function getTranslations()
             'filter_categories' => $this->translator->trans('Filter by categories', array(), 'Admin.Actions'),
             'filter_datepicker_from' => $this->translator->trans('From', array(), 'Admin.Global'),
             'filter_datepicker_to' => $this->translator->trans('To', array(), 'Admin.Global'),
+            'filter_low_stock' => $this->translator->trans('Display products under low stock level first', array(), 'Admin.Catalog.Feature'),
```

Comment:
Should be "Display products below low level first"
>> Should be "below" everywhere you have "under", please harmonize if others strings are concerned!
Ping @emmanuelf75009 who will confirm :)